.circleci/config.yml

```diff
@@ -280,16 +293,24 @@ jobs:
 # with limited amount of dependencies.
 workflows:
   version: 2
-  build_and_test:
+  draft_build:
```

Comment:
this gets run even if not in draft mode so we might want a different name to specify it